packages/account/pages/login.vue

```diff
@@ -57,21 +64,20 @@ export default {
 		}
 	},
 	methods: {
-		submit() {
+		async submit() {
 			this.hasError = false
 			if (this.$refs.login.validate()) {
 				this.$refs.login.resetValidation()
 				try {
-					const resp = this.$auth.login({
-						data: {
-							username: this.username,
-							password: this.password
-						}
+					await this.$auth.login({
+						username: this.username,
+						password: this.password
 					})
-					console.log(resp)
+					console.log(this.$auth)
```

Comment:
Console logging like this should be avoided. Probably best to have a eslint rule for this that.

Reply:
I forgot that when I was debugging. Going to remove it.

Reply:
If you use a proper debugger (like devtools), you don't have to add console logging like this.